src/Maker/MakeTwigComponent.php

```diff
@@ -47,20 +51,44 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
         ;
     }
 
-    public function configureDependencies(DependencyBuilder $dependencies): void
+    public function configureComposerDependencies(DependencyManager $dependencyManager): void
```

Comment:
I'm wondering if we can simplify. A few things:

* Could we keep `DependencyBuilder`? It already records the required dependencies, which allows us to know if we need to throw an error. Now we're just going to use this to run `composer require` instead of throwing an error.
* In `DependencyBuilder`, we have this idea of an "optional dependency". I'm wondering if we can scrap that. It seems rarely used and the usages look questionable. For example, in `MakeForm`, we mark `validator` as optional. We should be more opinionated: validation is required. We also mark `orm` as optional in `MakeForm`. That probably *should* be optional. But why record that as an optional dependency at all? I'd just not mention it or handle it at all.

The tl;dr is 

A) Can we keep using `DependencyBuilder` but not worry about this optional dependencies idea anymore?
B) For packages that we discover that we need only during the interactive phase (e.g. `ux-live-component` after answering the "live" question), could we just make some new service available (that maker commands could DI into their constructor) that allows you to trigger the install? e.g.

```php
if ($input->getOption('live')) {
    $this->dependencyDownloader->installPackage('symfony/ux-live-component', $io);
}
```

This could interactively ask them if they want to install and do it. Else throw an exception to exit the command. Or maybe we go further and just `composer require` packages that are needed and just notify the user as we're doing it.